src/signer.ts

```diff
@@ -408,6 +419,16 @@ export class URLSigner {
 
     return Math.round(expiresInMSeconds / 1000); // The API expects seconds.
   }
+
+  parseUsableFrom(usableFrom?: string | number | Date): number {
```

Comment:
Did you try refactoring `parseUsableFrom()` into `parseExpires()` they look really similar?